### DISCUSSION TITLE
it shouldnt be as easy to rotate the screen

### DISCUSSION THREAD

**Discussion by Wladefant** [personalizedrefrigerator/js-draw#45] [Ideas & Feature Requests]
I constantly rotate the notes, even though I do not want this. And honestly I dont even see the use case here in day to day live only in very specific use cases. I would suggest adding a rotation lock. For example like samsung notes does with the canvas lock
![SmartSelect_20231026_151244_Samsung Notes](https://github.com/personalizedrefrigerator/js-draw/assets/79611093/6d975166-0214-47e9-a479-de3302940f60)

**Comment by personalizedrefrigerator** [comment]
Try enabling rotation lock:
![screenshot: rotation lock under hand tool menu enabled](https://github.com/personalizedrefrigerator/js-draw/assets/46334387/81b7fdda-7d1a-4d6a-908a-6b589ac65b1e)

It might make sense to have this enabled by default.

**Reply by personalizedrefrigerator** [reply]
I've also just [more-than doubled the initial angle required to start rotation](https://github.com/personalizedrefrigerator/js-draw/blob/3e8127b9f2815a75e6cd1357c94485fbfd30ac9d/packages/js-draw/src/tools/PanZoom.ts#L96). I'll release a new APK soon with the changes.

**Edit**: [See this release](https://github.com/personalizedrefrigerator/joplin/releases/tag/prerelease-1).

**Reply by Wladefant** [reply]
Alright have not seen the lock rotation. That is perfect
